ref/README.md

```diff
@@ -26,6 +26,7 @@ The following packages are required for building and running this kernel:
 The following packages are optional for building and running this kernel:
 
 * [GPTL](https://github.com/jmrosinski/GPTL) (General Purpose Timing Library)
+* MPI
```

Comment:
Should we keep this bullet consistent with the one above and provide a link and description? Something similar to : 
* [MPI](https://www.open-mpi.org/) (Open-MPI, Message Passing Interface)

Reply:
@robgonzalezpita - There are many different MPI implementations (mpich, OpenMPI, Intel MPI, mvapich2, etc).  Any one of those will work if installed correctly.  Some are free, some require licenses.  Since there's no one definitive place to link and most people in our field know what `MPI` is, I feel this is sufficient.